src/main/kotlin/com/seokjoo/todo/domain/repository/todo/TodoRepository.kt

```diff
@@ -1,14 +1,18 @@
 package com.seokjoo.todo.domain.repository.todo
 
 import com.seokjoo.todo.domain.entity.todo.Todo
-import org.springframework.data.domain.Page
 import org.springframework.data.domain.Pageable
+import org.springframework.data.domain.Slice
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
 import org.springframework.stereotype.Repository
 
 @Repository
 interface TodoRepository : JpaRepository<Todo, Long> {
-    @Query("select t from Todo t left join fetch t.todoCategories tc left join fetch tc.category ORDER BY t.createdAt ASC")
-    fun findAllByOrderByCreatedAtAsc(pageable: Pageable): Page<Todo>
+    @Query("select t from Todo t ORDER BY t.createdAt ASC")
+    fun findAllSlicedTodo(pageable: Pageable): Slice<Todo>
```

Comment:
```suggestion
    fun findAllSlicedTodoOrderByCreatedAt(pageable: Pageable): Slice<Todo>
```

요런 네이밍 어떠신가요!
지금은 정렬해주는지 아닌지 이름만 봐서는 모를 거 같아요

Reply:
조아요 !!!